src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,50 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("Starting the process of completing an order for userId: {}", userId);
+        List<Product> products = null;
+        logger.info("Retrieving products from the shopping cart");
+        try {
+            products = getAllProductsFromShoppingCart(userId);
+            logger.debug("Products retrieved from shopping cart for userId: {}: {}",
+                    userId, products);
+        } catch (Exception e) {
+            logger.error("Failed to retrieve products from shopping cart for userId: {}",
+                    userId, e);
+            throw new RuntimeException("Please try again");
+        }
+
         Order order = new Order(products, userId);
+        logger.debug("Order created for userId: {} with products: {}", userId, products);
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.debug("Order ID set to {} for userId: {}", order.getOrderId(), userId);
+
+        logger.info("Order {} successfully completed for userId: {}",
+                order.getOrderId(), userId);
         return order;
     }
 
     private List<Product> getAllProductsFromShoppingCart(Long userId) {
         // NOTE: In production ready code this method should fetch data from DB
         // For test purpose we simplify this method and return dummy data
         Product iphone = new Product("iPhone X", BigDecimal.valueOf(1199));
+        logger.debug("Product iphone created for userId: {}", userId);
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
+        logger.debug("Product macBook created for userId: {}", userId);
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
+        logger.debug("Product xiaomi created for userId: {}", userId);
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Data successfully fetched from DB");
```

Comment:
The log message 'Data successfully fetched from DB' is misleading because there is no database interaction in the provided method. Please ensure log messages accurately reflect the actions being performed.